keps/sig-apps/2307-job-tracking-without-lingering-pods/README.md

```diff
@@ -380,112 +408,105 @@ _This section must be completed when targeting alpha to a release._
     - Will enabling / disabling the feature require downtime or reprovisioning
       of a node? (Do not assume `Dynamic Kubelet Config` feature is enabled).
 
-* **Does enabling the feature change any default behavior?**
+#### Does enabling the feature change any default behavior?
 
   Yes.
   
   - Removing terminated Pods doesn't affect Job status.
   - Pods removed by the user or other controllers count towards failures or
     completions.
 
-* **Can the feature be disabled once it has been enabled (i.e. can we roll back
-  the enablement)?**
+#### Can the feature be disabled once it has been enabled (i.e. can we roll back
+  the enablement)?
   
   Yes.
   The job controller removes finalizers in this case.
   Since some succeeded Pods might have been removed, the job controller will
   create new Pods to fulfill completions. But this is no different from
   existing behavior with the legacy tracking.
 
-* **What happens if we reenable the feature if it was previously rolled back?**
+#### What happens if we reenable the feature if it was previously rolled back?**
 
   Existing Jobs are tracked with legacy Pods.
 
-* **Are there any tests for feature enablement/disablement?**
+#### Are there any tests for feature enablement/disablement?**
 
   Yes, we plan to add integration tests.
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
-
-* **How can a rollout fail? Can it impact already running workloads?**
+#### How can a rollout fail? Can it impact already running workloads?
   Try to be as paranoid as possible - e.g., what if some components will restart
    mid-rollout?
+  
+  The change doesn't affect running Pods. If the component restarts
+  mid-rollout into an older version, the Job controller switches to tracking
+  Jobs without using finalizers.
 
-* **What specific metrics should inform a rollback?**
-
-* **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+#### What specific metrics should inform a rollback?
 
-* **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
-fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  - An increase in `job_sync_duration_seconds`. Users should expect a higher
+    duration than previous versions of the job controller due to the new API
+    calls.
+  - Stale `job_sync_total` or `job_finished_total`.
 
-### Monitoring Requirements
+#### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+  
+  Integration tests cover feature gate disablement.
+  A manual upgrade->downgrade->upgrade flow will be executed to ensure that a
```

Comment:
Can you please ack that those will get run before feature will graduate? And that the KEP will get updated with their results [sth like: https://github.com/kubernetes/enhancements/pull/2538 ]

Reply:
Ack